styles/styles.css

```diff
@@ -498,6 +498,12 @@ main .block {
   margin-right: auto;
 }
 
+main .block.border {
+  border: 1px solid;
+  padding: 0.5rem;
+  margin-bottom: 1rem;
```

Comment:
Probably these are applicable only when there's a `text` box with border, so should move to `text` block. The `border` style should ideally just impose a border.

Reply:
Ok got it